### PR TITLE
chore(client): Add the strings needed for the email reconfirmation feature.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@vbudhram - r?

PR #3651
mozilla/fxa-auth-mailer#141

### CODE REVIEW

File: app/scripts/lib/strings.js [hunk 1]
```diff
@@ -25,6 +25,27 @@ define(function (require, exports, module) {
...
+  // Strings for "reconfirm your email", fxa-83
+  t('Confirm new sign-in to Firefox');
+  t('You signed in to Firefox from %(device)s, %(location)s.');
+  t('You signed in to Firefox from %(device)s.');
+  t('For added security, please confirm this sign-in to begin syncing with %(device).');
+  t('Alternatively:');
+  t('This is an automated email; if you received it in error, no action is required.');
+  t('For more information, please visit <a %(supportLinkAttributes)s>Mozilla Support</a>');
+  t('Verify Email');
+  t('Verify your email to finish your Firefox Account sign-in');
```
Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: @vbudhram - I replaced `registration` with `sign-in` here.

File: app/scripts/lib/strings.js [hunk 2]
```diff
@@ -25,6 +25,27 @@ define(function (require, exports, module) {
   // for why this could be necessary.
   t('<a href="https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account">Help</a>');
 
+  // Strings for "reconfirm your email", fxa-83
+  t('Confirm new sign-in to Firefox');
+  t('You signed in to Firefox from %(device)s, %(location)s.');
+  t('You signed in to Firefox from %(device)s.');
```
Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: I added both strings to give us flexibility, one with location, one without.

File: app/scripts/lib/strings.js [hunk 3]
```diff
@@ -25,6 +25,27 @@ define(function (require, exports, module) {
...
+  t('For added security, please confirm this sign-in to begin syncing with %(device).');
```
Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: I added `For added security, ` to this string to go along with the mocks.

**Comment by vbudhram** [comment]
Looked over [mockups](https://github.com/mozilla/fxa/blob/phil/fxa-83/features/FxA-83-sign-in-confirmation-email/README.md) and strings, LGTM, 👍

**Pull request merged**